package.json

```diff
@@ -37,6 +38,42 @@
   "dependencies": {
     "body-parser": "^1.16.0",
     "cookie-parser": "^1.4.3",
-    "express": "^4.14.0"
+    "express": "^4.14.0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-redux": "^5.0.2",
+    "react-router": "^3.0.2",
+    "react-router-redux": "^4.0.7",
```

Comment:
I'm still a big supporter of redux-little-router, although I have yet to implement it on a project. Ryan said he had trouble figuring it out. Have you worked with it before?